logs/file.go

```diff
@@ -53,7 +54,8 @@ type fileLogWriter struct {
 
 	Level int `json:"level"`
 
-	Perm os.FileMode `json:"perm"`
+	Perm string `json:"perm"`
+	perm os.FileMode
```

Comment:
Perm just change to string, but I don't think we need another param perm. just convert it to os.FileMode in `createLogFile`